scss/components/_button.scss

```diff
@@ -294,7 +294,7 @@ $button-responsive-expanded: false !default;
 @mixin button-dropdown(
   $size: 0.4em,
   $color: $white,
-  $offset: get-side($button-padding, right)
+  $offset: $offset
```

Comment:
I think that we need to create a new variable for the default offset so that it can be referenced. 

```
$dropdown-offset : get-side($button-padding, right);

@mixin button-dropdown(
  $size: 0.4em,
  $color:white,
  $offset: $dropdown-offset
) {
```

Reply:
I think I misunderstood the initial issue. We can close this as this parameter is already an optional one and the parameter is correctly passed.